hack/tools/release/notes.go

```diff
@@ -403,8 +446,82 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 
 	fmt.Println("")
 	fmt.Println("_Thanks to all our contributors!_ 😊")
+}
 
-	return 0
+func parseWeeklyUpdate(commits []*commit, merges map[string][]string, commitRange string, lastDay string) {
+	for _, c := range commits {
+		body := trimTitle(c.body)
+		var key, prNumber, fork string
+		switch {
+		case strings.HasPrefix(body, ":sparkles:"), strings.HasPrefix(body, "✨"):
+			key = features
+			body = strings.TrimPrefix(body, ":sparkles:")
+			body = strings.TrimPrefix(body, "✨")
+		case strings.HasPrefix(body, ":bug:"), strings.HasPrefix(body, "🐛"):
+			key = bugs
+			body = strings.TrimPrefix(body, ":bug:")
+			body = strings.TrimPrefix(body, "🐛")
+		case strings.HasPrefix(body, ":book:"), strings.HasPrefix(body, "📖"):
+			key = documentation
+			body = strings.TrimPrefix(body, ":book:")
+			body = strings.TrimPrefix(body, "📖")
+			if strings.Contains(body, "CAEP") || strings.Contains(body, "proposal") {
+				key = proposals
+			}
+		case strings.HasPrefix(body, ":seedling:"), strings.HasPrefix(body, "🌱"):
+			key = other
+			body = strings.TrimPrefix(body, ":seedling:")
+			body = strings.TrimPrefix(body, "🌱")
+		case strings.HasPrefix(body, ":warning:"), strings.HasPrefix(body, "⚠️"):
+			key = warning
+			body = strings.TrimPrefix(body, ":warning:")
+			body = strings.TrimPrefix(body, "⚠️")
+		default:
+			key = unknown
+		}
+
+		body = strings.TrimSpace(body)
+		if body == "" {
+			continue
+		}
+		body = fmt.Sprintf("\t - %s", body)
+		_, _ = fmt.Sscanf(c.merge, "Merge pull request %s from %s", &prNumber, &fork)
+		if key == documentation {
+			merges[key] = append(merges[key], prNumber)
+			continue
+		}
+		merges[key] = append(merges[key], formatMerge(body, prNumber))
+	}
+
+	// TODO Turn this into a link (requires knowing the project name + organization)
+	fmt.Println("Weekly update :rotating_light:")
+	fmt.Printf("Changes from %v a total of %d new commits were merged into milestone %s.\n\n", commitRange, len(commits), *milestone)
+
+	for _, key := range outputOrder {
+		mergeslice := merges[key]
+		if len(mergeslice) == 0 {
+			continue
+		}
+
+		switch key {
+		case documentation:
+			fmt.Printf("- %d Documentation and book contributions :book: \n\n", len(mergeslice))
+		case other:
+			fmt.Printf("- %d Other changes :seedling:\n\n", len(merges[other]))
+		default:
+			fmt.Printf("- %d %s %s\n", len(merges[key]), key, emoteMap[key])
+			for _, merge := range mergeslice {
+				fmt.Println(merge)
+			}
+			fmt.Println()
+		}
+	}
+
+	fmt.Println("All merged PRs can be viewed in GitHub:")
+	fmt.Println("https://github.com/kubernetes-sigs/cluster-api/pulls?q=is%3Apr+closed%3A" + *since + ".." + lastDay + "+is%3Amerged+milestone%3A" + *milestone + "+\n")
```

Comment:
```suggestion
	fmt.Println("https://github.com/kubernetes-sigs/cluster-api/pulls?q=is%3Apr+closed%3A" + *since + ".." + lastDay + "+is%3Amerged+milestone%3A" + *milestone + "+\n")
```

This isn't portable to other projects. That's probably fine as this is only used by CAPI today, butit might be worth highlighting that somewhere

Reply:
You could also take the project URL as an arg, this might also help with the above TODO.